requirements/readthedocs.txt

```diff
@@ -1,3 +1,18 @@
+imgaug
+kwarray
+lmdb
+matplotlib
 mmcv
+numpy
+Pillow<=6.2.2
+Polygon3
+pyclipper
+pycocotools
+python-Levenshtein
+scikit-image
+scipy
+shapely
+skimage
+titlecase
```

Comment:
There is no need to install all dependencies in readthedocs. Most ones can be mocked.